internal/hammer/ddl.go

```diff
@@ -57,8 +57,10 @@ func (u Update) defaultValue() string {
 		return "false"
 	case spansql.Int64, spansql.Float64:
 		return "0"
-	case spansql.String, spansql.Bytes:
+	case spansql.String:
 		return "''"
+	case spansql.Bytes:
+		return "CAST('' AS BYTES)"
```

Comment:
Is it possible to use byte literals?

https://cloud.google.com/spanner/docs/lexical#string-and-bytes-literals

Reply:
thanks!
I'll  fix it.